app/api/crud.py

```diff
@@ -451,14 +451,22 @@ async def get_terms(
                 ),
                 None,
             )
-            term_label_dicts.append(
```

Comment:
Similar to my comment above, now that we don't always return just the `TermURL` and `Label` anymore, maybe we can make this variable name more generic. Maybe something like `term_metadata` instead of `term_label_dicts`?